### DISCUSSION TITLE
drawFromSVG

### DISCUSSION THREAD

**Discussion by alzatin** [sgenoud/replicad#158] [General]
Hi! I saw in an old discussion that you had been planning to add a drawfromSVG to the library, any chance you are still working on that? It would be really great and super helpful to be able to import an SVG!

**Comment by sgenoud** [comment]
Hei,

You can import if from the [replicad-decorate](https://github.com/sgenoud/replicad-decorate/tree/main) library. I feel that, while drawing from an SVG is very useful, it is not always very well defined (and prone to errors). Do you have some issues with importing from another library?

**Reply by alzatin** [reply]
No problem with that. I will give it a try! Thank you. Also thank you for alerting me to this library, I had not run across it and I think I'll find it useful!
